dev/PushNotifications/PushNotificationsLongRunningTask/platform.h

```diff
@@ -0,0 +1,40 @@
+﻿#pragma once
+
+#include "../PushNotifications-Constants.h"
+
+struct __declspec(uuid(PUSHNOTIFICATIONS_IMPL_CLSID_STRING)) NotificationsLongRunningPlatformImpl WrlFinal :
+Microsoft::WRL::RuntimeClass<
+    Microsoft::WRL::RuntimeClassFlags<Microsoft::WRL::ClassicCom>,
+    INotificationsLongRunningPlatform,
+    Microsoft::WRL::FtmBase>
+{
+    void Initialize();
```

Comment:
Do all these methods throw? If not, please mark those that don't throw with noexcept.